CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# How to Contribute
+
+Thank you for taking your time to consider contributing to the repository!
+
+To get started, follow the instructions in this document.
+
+## Report a bug fix
+
+To report a bug fix, follow the issue template found when openning a new issue, if possible and available attach a [codesandbox](https://codesandbox.io/) or [jsBin](https://codesandbox.io/) with the error being reported.
+
+## Propose a Feature
+
+Open a issue, mark as other, and describe the current scenario, it's problems, and how your resource solves a problem or introduce new functionalities.
+
+## Choose a Task on the Project
+
+Choose a task in the [project](https://github.com/agrotis-io/vraw/projects/1), read the description of the issue and if you have any questions, leave a comment.
+
+## Issue Labeling
+
+To identify what kind of issue you are planning to work, or to check our list of tasks and issues in more detail take a look at this table:
+
+| Label            | Description                                                        |
+| ---------------- | ------------------------------------------------------------------ |
+| docs             | Indicate documentation development necessary                       |
+| feat             | Indicate feature development                                       |
+| CI               | Indicate Continuous integration development                        |
+| refactor         | Indicate code refactor                                             |
+| remove           | Indicate feature removal                                           |
+| fix              | Indicates an unexpected problem or unintended behavior             |
+| duplicate        | Indicates similar issues or pull requests                          |
+| enhancement      | Indicates new feature requests                                     |
+| good first issue | Indicates a good issue for first-time contributors                 |
+| help wanted      | Indicates that a maintainer wants help on an issue or pull request |
+| invalid          | Indicates that an issue or pull request is no longer relevant      |
+| question         | Indicates that an issue or pull request needs more information     |
+| wontfix          | Indicates that work won't continue on an issue or pull request     |
+
+## Fork or Clone It
+
+In order to start colaborating to our project, you have to **FORK** this repository if you are a contributor, or **CLONE** it if you are a not a contributor.
+
+## Development Workflow
+
+This will help you understand how you should develop your contribution to the repository:
```

Comment:
get the link for slides of this workflow, even with the image it is complicated to understand each step